client/src/featureform/register.py

```diff
@@ -1701,11 +1714,15 @@ def get_label(self, name, variant=None):
         Returns:
             label (Union[label, LabelVariant]): Label or LabelVariant
         """
+        if local:
```

Comment:
Can change this to
if local and not variant:
elif local and variant:
elif not local and not variant
elif not local and variant
To void the nesting

Reply:
i thought this would make it easier for when we decide to refactor the client code into localclient and hostclient impls